.teamcity/.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,33 @@
+
+
+<!--
+  ~ Copyright 2022 the original author or authors.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<gradleEnterprise
+        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+    <server>
+        <url>https://e.grdev.net</url>
```

Comment:
The main GBT build uses `ge.gradle.org` so that everyone (even outside Gradle team) can view the build scans. I suppose we want to do the same thing for Maven builds, right?

Reply:
I wasn't sure, I thought we might want to keep the TC configuration info more private in the future.  I know it's part of the public `gradle/gradle` project right now, so it isn't, but @big-guy and I had briefly talked about moving it.

Do you think anyone outside of the Gradle team would have any interest in these?

Reply:
I would err on the side of sending these to e.grdev.net first.  I think we don't know what the output looks like from these

Reply:
OK, let's use `e.grdev.net` for now.